### PR TITLE
Add `osu!mania 4K World Cup 2024: Round of 32 Recap` news post

### PR DESCRIPTION
**Pull request by 0x84f**
🎹

[preview](https://osu2.roanh.dev/home/news/2024-08-28-osumania-4k-world-cup-2024-round-of-32)
needs re-time

### CODE REVIEW

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+- We all know ::{ flag=CL }:: [myucchi](https://osu.ppy.sh/users/10072733) and their feats. While their performance across the entire mappool [vs. New Zealand](https://osu.ppy.sh/community/matches/115223051) was phenomenal, they crushed it in [Mono. - Tricolor Prizm☆☆☆ミ](https://osu.ppy.sh/beatmapsets/2024637#mania/4217619) with a score of 997,781.
+- Taking everyone by surprise, ::{ flag=VN }:: [Joki_](https://osu.ppy.sh/users/18043326) has let everybody know they're a force to be reckoned with. Their rice performance was second to none, as seen with their play on [SHRINE 418 - Little Little Princess](https://osu.ppy.sh/beatmapsets/2234579#mania/4746363), putting up a score of 997,904 [during their match vs. Ukraine](https://osu.ppy.sh/community/matches/115210872).
+- ::{ flag=JP }:: [CrewK](https://osu.ppy.sh/users/11488604) demonstrated yet again who's king, with an extraterrestrial performance on [Shpongle - Outer Shpongolia](https://osu.ppy.sh/beatmapsets/2234598#mania/4746405) during the Japan vs. France match, scoring an astonishing 998,578 points.
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:117]: either hyphen or colon, not em dash
i'd also include the artist

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+[**You can view the match information here!**](https://osu.ppy.sh/community/matches/115212102)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2233665337&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Indonesia's renewed roster with 6 new faces had an interesting first match against the higher-seeded Hong Kong. If you don't feel like watching the whole match, these highlights should contain the gist:
+
+- [Little Little Princess](https://osu.ppy.sh/beatmapsets/2234579#mania/4746363) (RC3) @ [15:40](https://www.twitch.tv/videos/2233665337?t=00h15m40s) — The first breakpoint of the match, a cohesive triple way FC from Indonesia to take the match lead.
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:88]:
```suggestion
- [Little Little Princess](https://osu.ppy.sh/beatmapsets/2234579#mania/4746363) (RC3) @ [15:40](https://www.twitch.tv/videos/2233665337?t=00h15m40s) — The first breakpoint of the match, a cohesive three-way FC from Indonesia to take the match lead.
```

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+### ::{ flag=SG }:: Singapore vs. ::{ flag=PL }:: Poland
+
+[**You can view the match information here!**](https://osu.ppy.sh/community/matches/115212044)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2233674722&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Singapore vs. Poland was a very interesting match to say nevertheless. While most viewers were expecting Singapore to sweep the match, especially with how far both teams' seeding were being #11 and #22 respectively, viewers couldn’t have been more wrong for what they were about to witness on the world cup stage. If you don't feel like watching the whole match, these highlights should contain the gist:
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:102]:
```suggestion
Singapore vs. Poland was a very interesting match to say the least. While most viewers were expecting Singapore to sweep the match, especially with how far both teams' seeding were being #11 and #22 respectively, viewers couldn’t have been more wrong for what they were about to witness on the world cup stage. Again, feel free to check the highlights below (or binge the entire match):
```

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+- [Ignite My Heart](https://osu.ppy.sh/beatmapsets/2234600#mania/4746408) (SV1) @ [10:50](https://www.twitch.tv/videos/2233674722?t=00h10m50s) — The first breakpoint of the match, an insane performance from Poland by taking the point of Singapore’s protected pick, winning by 3,480 score.
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:104]:
```suggestion
- [Ignite My Heart](https://osu.ppy.sh/beatmapsets/2234600#mania/4746408) (SV1) @ [10:50](https://www.twitch.tv/videos/2233674722?t=00h10m50s) — The first breakpoint of the match. An insane performance from Poland by taking the point of Singapore’s protected pick, winning by 3,480 score.
```

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+- [CODE:0](https://osu.ppy.sh/beatmapsets/1804396#mania/3760851) (RC3) @ [19:05](https://www.twitch.tv/videos/2233674722?t=00h19m05s) — Another breakpoint, this time from Singapore with all team members getting a score v1 SS.
+- [Hallucinate (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2234584#mania/4746373) (RC2) @ [37:22](https://www.twitch.tv/videos/2233674722?t=00h37m22s) — Yet another breakpoint, this time from Poland with a miss from Singapore at the very end, letting them barely get the point.
+- [Senkou no Flare (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2234576#mania/4746352) (RC5) @ [42:28](https://www.twitch.tv/videos/2233674722?t=00h42m28s) — Poland continued their momentum from the last pick and tied the match up 3–3 with each team member putting up insane scores while viewers were blown away.
+
+Unfortunately for Poland, Singapore managed to clutch up and take the match 5–3. While Singapore took the match this time around, the Polish will not be forgotten with their incredible performance and beating everyone’s expectations. We wish Poland the very best going into the losers' bracket of the tournament.
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:109]:
```suggestion
Unfortunately for Poland, Singapore managed to clutch up and take the match 5–3. While Singapore took the match this time around, the Polish will not be forgotten with their incredible performance beating everyone’s expectations. We wish Poland the very best going into the losers' bracket of the tournament.
```

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+## Score highlights
+
+Earlier rounds are known to be calmer as higher seeded teams get matched against their lower seeded peers. Even so, attention should be brought to a couple of note-worthy scores and players to keep track of:
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:113]:
```suggestion
Earlier rounds are known to be calmer as higher-seeded teams get matched against their lower-seeded peers. Even so, attention should be brought to a couple of note-worthy scores and players to keep track of:
```

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md (hunk 1 above)
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:117]: maybe link jp vs fr match like the others?

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+
+- We all know ::{ flag=CL }:: [myucchi](https://osu.ppy.sh/users/10072733) and their feats. While their performance across the entire mappool [vs. New Zealand](https://osu.ppy.sh/community/matches/115223051) was phenomenal, they crushed it in [Mono. - Tricolor Prizm☆☆☆ミ](https://osu.ppy.sh/beatmapsets/2024637#mania/4217619) with a score of 997,781.
+- Taking everyone by surprise, ::{ flag=VN }:: [Joki_](https://osu.ppy.sh/users/18043326) has let everybody know they're a force to be reckoned with. Their rice performance was second to none, as seen with their play on [SHRINE 418 - Little Little Princess](https://osu.ppy.sh/beatmapsets/2234579#mania/4746363), putting up a score of 997,904 [during their match vs. Ukraine](https://osu.ppy.sh/community/matches/115210872).
+- ::{ flag=JP }:: [CrewK](https://osu.ppy.sh/users/11488604) demonstrated yet again who's king, with an extraterrestrial performance on [Shpongle - Outer Shpongolia](https://osu.ppy.sh/beatmapsets/2234598#mania/4746405) during the Japan vs. France match, scoring an astonishing 998,578 points.
+
+Keep a look-out for these people! Let's see how future rounds fare them, and whether they can keep up the pace.
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:119]:
```suggestion
Keep a lookout for these people! Let's see how future rounds fare them, and whether they can keep up the pace.
```

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+
+---
+
+Now is the perfect time to start following the MWC4K, choose your favourites and cheer them on!
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:123]:
```suggestion
Now is the perfect time to start following the MWC4K, choose your favourite teams and cheer them on!
```

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+
+Regardless of whom you'll be cheering on, we hope to see you this weekend for the Round of 16 on [Twitch](https://www.twitch.tv/osulive), and if you are interested in the nitty-gritty of this MWC4K edition, you can find the full details over on the [wiki page](/wiki/Tournaments/MWC/2024_4K)!
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:125]: i believe this is fairly important to always mention. linking the spreadsheet works too for this purpose if i'm not mistaken

```suggestion
Regardless of whom you'll be cheering on, we hope to see you this weekend for the Round of 16 on [Twitch](https://www.twitch.tv/osulive), and if you are interested in the nitty-gritty of this MWC4K edition (including match schedules), you can find the full details over on the [wiki page](/wiki/Tournaments/MWC/2024_4K)!
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2233665337&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Indonesia's renewed roster with 6 new faces had an interesting first match against the higher-seeded Hong Kong. If you don't feel like watching the whole match, these highlights should contain the gist:
+
+- [Little Little Princess](https://osu.ppy.sh/beatmapsets/2234579#mania/4746363) (RC3) @ [15:40](https://www.twitch.tv/videos/2233665337?t=00h15m40s) — The first breakpoint of the match, a cohesive triple way FC from Indonesia to take the match lead.
+- [Brain Rigid Girl](https://osu.ppy.sh/beatmapsets/2234595#mania/4746400) (LN2) @ [41:05](https://www.twitch.tv/videos/2233665337?t=00h41m05s) — A clutch save by Hong Kong on match point, and the closest pick in this entire RO32!
+- [Hebe](https://osu.ppy.sh/beatmapsets/2234586#mania/4746375) (HB2) @ [52:18](https://www.twitch.tv/videos/2233665337?t=00h52m18s) — The tiebreaker, the essence of all tournaments, the final consistency test for Hong Kong to secure their victory.
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:90]: either hyphen or colon, not em dash

i'd also include the artist

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md (hunk 9 above)
Comment (0x84f) [reply news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:125]: everything's on the wiki so should be fine

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+date: 2024-08-28 12:00:00 +0000
+---
+
+The key smashing has officially commenced last weekend, and now is the perfect time to pull out the Round of 32 highlight reel.
+
+![](/wiki/Tournaments/MWC/2024_4K/img/mwc4k2024-banner.jpg)
+
+We all know qualifiers have never been the deciding factor in a tournament. While they definitely provide an initial edge to those teams who prove to go above and beyond the competition, qualifiers are but the start of the show!
+
+This year we've witnessed everything, with this year's qualifiers being one of the most competitive to date. From the usual teams getting podium placements, to fan favourites such as Japan losing some of its previous years' lustre, or even seeing new faces in higher seeds such as the United Kingdom!
```
Comment (Walavouchey) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:13]: the second sentence isn't complete

```suggestion
This year's qualifiers were one of the most competitive to date. From the usual teams getting podium placements, to fan favourites such as Japan losing some of their previous years' lustre, or even seeing new faces in higher seeds such as the United Kingdom, it seems this year we've witnessed just about everything!
```

Comment (0x84f) [reply news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md]: done in 9080ed36933fd600bbfb58bd84ff920ed2b6ebae

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md (hunk 10 above)
Comment (0x84f) [reply news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md]: done in 44eabff9847469a12aabba3da753132ff4d7c0f1

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md (hunk 1 above)
Comment (0x84f) [reply news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md]: yup, 44eabff9847469a12aabba3da753132ff4d7c0f1

File: news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2233665337&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Indonesia's renewed roster with 6 new faces had an interesting first match against the higher-seeded Hong Kong. If you don't feel like watching the whole match, these highlights should contain the gist:
+
+- [SHRINE 418 - Little Little Princess](https://osu.ppy.sh/beatmapsets/2234579#mania/4746363) (RC3) @ [15:40](https://www.twitch.tv/videos/2233665337?t=00h15m40s) - The first breakpoint of the match, a cohesive three-way FC from Indonesia to take the match lead.
+- [ShinRa-Bansho - Brain Rigid Girl](https://osu.ppy.sh/beatmapsets/2234595#mania/4746400) (LN2) @ [41:05](https://www.twitch.tv/videos/2233665337?t=00h41m05s) - A clutch save by Hong Kong on match point, and the closest pick in this entire RO32!
+- [SiLiS - Hebe](https://osu.ppy.sh/beatmapsets/2234586#mania/4746375) (HB2) @ [52:18](https://www.twitch.tv/videos/2233665337?t=00h52m18s) - The tiebreaker, the essence of all tournaments, the final consistency test for Hong Kong to secure their victory.
```
Comment (kadoen) [inline news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md:90]: Supposed to be

- [lapix - Flying Castle (Extended Mix)](https://osu.ppy.sh/beatmapsets/2234596#mania/4746402) (TB)

Comment (0x84f) [reply news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md]: 9372f371f81260997e9bfb068f64826e3d23c7d8

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**